statsmodels/stats/anova.py

```diff
@@ -362,6 +367,198 @@ def anova_lm(*args, **kwargs):
     return table
 
 
+class AnovaRM(object):
+    """
+    Repeated measures ANOVA using least square regression
+
+    The full model regression residual sum of squared is
+    used to compared with reduced model for calculating the
+    within subject effect sum of squared [1]
+
+    Between subject effect and Greenhouse-Geisser correction is not yet
+    supported.
```

Comment:
Both should be able to added without backwards compatible break.  I currently have no idea how to implement the support for between subject effect using the current regression based approach though.